src/open_prime_rando/version.py

```diff
@@ -1,4 +1,4 @@
 # file generated by setuptools_scm
 # don't change, don't track in version control
-__version__ = version = '0.6.2.dev0'
-__version_tuple__ = version_tuple = (0, 6, 2, 'dev0')
+__version__ = version = '0.6.2.dev8'
+__version_tuple__ = version_tuple = (0, 6, 2, 'dev8')
```

Comment:
?

Reply:
Oh, forgot to update gitignore when moving things around